src/logic/calculate.js

```diff
@@ -0,0 +1,36 @@
+import operate from './operate';
+
+const calculate = (data, name) => {
+  let { total, next, operation } = data;
+
+  if (name === '+/-') {
```

Comment:
### Heads-Up:
**we need to add more case;** 
1. if the `buttonName` = `A/C`, we should return `0`